tensorflow/core/grappler/optimizers/remapper_test.cc

```diff
@@ -925,5 +925,64 @@ TEST_F(RemapperTest, FuseConv2DWithSqueezeAndBias) {
 }
 #endif
 
+#define REGISTER_TEST_ALL_TYPES(TEST) \
+  REGISTER_TEST(TEST, DT_FLOAT);      \
+  REGISTER_TEST(TEST, DT_BFLOAT16);
+
+#define REGISTER_TEST(CMP, TYPE)                                              \
```

Comment:
Can you pull this test into a template function and reduce the size of the macro. Ideally I would not use macros at all, and just manually create all the test combinations, or at least leave only the function call inside the macro body.

Reply:
Done. Only register test for Equal and cast, since the other comparison ops are using exactly the same fusion logic.